test/ExportQueryExecutionTreeTest.cpp

```diff
@@ -175,19 +175,27 @@ TEST(ExportQueryExecutionTree, Integers) {
       kg, query, 3,
       // TODO<joka921> the ORDER BY of negative numbers is incorrect.
       // TSV
-      "?o\n"
-      "-42019234865781\n"
-      "42\n"
-      "4012934858173560\n",
+      "?o\t?completedWord\n"
+      "-42019234865781\t\n"
+      "42\t\n"
+      "4012934858173560\t\n",
       // CSV
-      "o\n"
-      "-42019234865781\n"
-      "42\n"
-      "4012934858173560\n",
-      makeExpectedQLeverJSON(
-          {"\"-42019234865781\"^^<http://www.w3.org/2001/XMLSchema#int>"s,
-           "\"42\"^^<http://www.w3.org/2001/XMLSchema#int>"s,
-           "\"4012934858173560\"^^<http://www.w3.org/2001/XMLSchema#int>"s}),
+      "o,completedWord\n"
+      "-42019234865781,\n"
+      "42,\n"
+      "4012934858173560,\n",
+      []() {
```

Comment:
These changes are not necessary if you add the word variable only if it is selected and only if there is a matching text record.